Mathlib/Analysis/SpecialFunctions/Integrals.lean

```diff
@@ -453,25 +484,49 @@ theorem integral_exp_mul_complex {c : ℂ} (hc : c ≠ 0) :
   · fun_prop
 
 @[simp]
-theorem integral_log (h : (0 : ℝ) ∉ [[a, b]]) :
-    ∫ x in a..b, log x = b * log b - a * log a - b + a := by
-  have h' := fun x (hx : x ∈ [[a, b]]) => ne_of_mem_of_not_mem hx h
-  have heq := fun x hx => mul_inv_cancel₀ (h' x hx)
-  convert integral_mul_deriv_eq_deriv_mul (fun x hx => hasDerivAt_log (h' x hx))
-    (fun x _ => hasDerivAt_id x) (continuousOn_inv₀.mono <|
-      subset_compl_singleton_iff.mpr h).intervalIntegrable
-        continuousOn_const.intervalIntegrable using 1 <;>
-  simp [integral_congr heq, mul_comm, ← sub_add]
-
-@[simp]
-theorem integral_log_of_pos (ha : 0 < a) (hb : 0 < b) :
```

Comment:
Can you *not* simply delete these lemmas please, but deprecate them instead? This way, somebody using them in a project depending on mathlib won't get errors "huh, my lemma is gone", but a warning "use lemma X isntead".

Write something like the following, right before the lemma to be deprecated
```
[deprecated (since := "2025-01-12")]
alias oldLemmaName := yourBetterReplacement

Reply:
Done, following the suggestion by @joneugster 
Thanks again.